app/src/main/java/org/jellyfin/androidtv/ui/browsing/MainActivity.kt

```diff
@@ -158,6 +160,21 @@ class MainActivity : FragmentActivity() {
 
 		NavigationAction.GoBack -> supportFragmentManager.popBackStack()
 
+		NavigationAction.GoHome -> {
```

Comment:
It might be better to re-use the NavigateFragment action but add a new "clear" flag to it instead of adding an entire new action.